src/main/kotlin/io/snyk/plugin/ui/UIUtils.kt

```diff
@@ -84,12 +84,16 @@ fun insertTitleAndResizableTextIntoPanelColumns(
     )
 }
 
-fun snykCodeAvailabilityPostfix(): String = when {
-    !isSnykCodeAvailable(pluginSettings().customEndpointUrl) -> " (disabled for endpoint)"
-    (pluginSettings().sastOnServerEnabled ?: true && pluginSettings().localCodeEngineEnabled ?: true) ->
-        " (disabled due to Local Code Engine)"
-    !(pluginSettings().sastOnServerEnabled ?: false) -> " (disabled in Snyk.io)"
-    else -> ""
+fun snykCodeAvailabilityPostfix(): String {
+    val sastOnServerEnabled = pluginSettings().sastOnServerEnabled
+    val localCodeEngineEnabled = pluginSettings().localCodeEngineEnabled
+    return when {
+        !isSnykCodeAvailable(pluginSettings().customEndpointUrl) -> " (disabled for endpoint)"
+        (sastOnServerEnabled == null && localCodeEngineEnabled == null) -> " (disabled due to unreachable server settings)"
```

Comment:
better `undefined` settings?

Reply:
Unreachable is a bit more correct word here. As undefined could mean we do reach server but such settings been undefined there, which is separate case.